### PR TITLE
fix: correct fee rate in tx detail

### LINKED ISSUE #3355: Difference in fee rate between transaction detail modal and send form
**Describe the bug**
Fee rate shown in Transaction details modal is different from the cusom fee filled in send form (and shown in review modal)
**Info:**
 - Suite Version: fa056457fde7042c06cdb75ef7772243942a5433

**Problem**: fee calculation based on tx.size instead of vsize. 
Prerequisite: https://github.com/trezor/blockbook/issues/559 (We'll wait for implementation of vsize field on the backend side of things)

Screenshot 2021-02-08 at 11 01 00
Screenshot 2021-02-08 at 11 01 14
Screenshot 2021-02-08 at 11 01 25

### LINKED ISSUE #5544: Display virtual bytes for segwit accounts
For segwit accounts calculate vsize and display rate in sat/vB

Related to: https://github.com/trezor/trezor-suite/issues/3355

### PR DESCRIPTION
**Pull request by mroz22**
We can finally resolve #3355 (specifically https://github.com/trezor/trezor-suite/issues/3355#issuecomment-1240449933) 

Changes: 
- `size` seems to be available from backends, doesnt need to be calculated locally anymore. Not avaialble everywhere though, so fallback kept in place. 
- added `tx.vsize` field which is now available from backends. Not everywhere, fallback. 
- added `tx.feeRate` field

Notes: 
- I was considering `tx.feeRate` or `tx.details.feeRate` option. But decided for the first one, details should be imho removed in some future version as it is concept that only suite is using and blockchain-link does not need to know about that. 
- As I mentioned, there are fallbacks and there are 2 reasons for them. Not all backends have been updated yet. This one could be solved and fallback thus removed. But we also need to cover transactions saved in users local state. 
- We could consider migration but we would either need to A] refetch data during migration which is imho NO-GO, B] compute vsize from data we already have but it would be quite heavy-weight and I am not sure if doable, maybe yes. My current gut feeling is that it is not worth doing it.
- I am not sure about this newly added test fixture [e4aa0d9](https://github.com/trezor/trezor-suite/pull/6261/commits/e4aa0d98b7ffbc4abaf3c4d1b82f80a3f31155da) it has so many unrelated and unreal fields. I don't like it very much :thinking: 

Commits: 

QA: 
- please check transactions list for all btc like coins. fee rate in tx detail should match fee rate in blockbook. some blockbooks might not have this new feature deployed yet.
- try rbf transaction for btc and testnet
- altcoins should not break (eth, ada, ripple)

Related issues: 
-  https://github.com/trezor/blockbook/issues/559
- part of https://github.com/trezor/trezor-suite/issues/3327

### CODE REVIEW

**Comment by matejkriz** [comment]
no need for transaction migration?

**Comment by matejkriz** [comment]
partially resolve https://github.com/trezor/trezor-suite/issues/3327?

**Comment by mroz22** [comment]
> no need for transaction migration?

I guess we need to keep the legacy way in place (as a fallback) :( since we can't really migrate this.

File: packages/blockchain-link/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 2.1.5
+
+-   added Transaction.feeRate
+-   use Transaction.size provided by Blockbook instead of computing it from hex if
```
Comment (marekrjpolak) [inline packages/blockchain-link/CHANGELOG.md]: `if` what? :)

**Review by marekrjpolak — APPROVED** [review]
LGTM

File: packages/blockchain-link/CHANGELOG.md (hunk 1 above)
Comment (mroz22) [reply packages/blockchain-link/CHANGELOG.md]: forcepushed fix

**Pull request merged**